tests/async-unit/test_appeal_fan_in.py

```
@@ -0,0 +1,81 @@
+from concurrent.futures import Future, ThreadPoolExecutor
+
+from fighthealthinsurance.generate_appeal import (
+    GeneratedAppeal,
+    _appeals_as_futures_complete,
+)
+from fighthealthinsurance.utils import UnwrapIterator
+
+
+def _mapper(name, future):
+    """Minimal stand-in for _generated_to_appeals_text: wait, then map."""
+    for kind, text in future.result():
```

Ruff (0.16.0)

[warning] Loop control variable `kind` not used within loop body

Rename unused `kind` to `_kind`

(B007)